play/PlayerRules/Basic Rules.txt

```diff
@@ -521,6 +565,51 @@ Hunkering costs one action, and you cannot fire your weapons. Imagine
 curling up in a ball, to pose the absolute smallest target possible. Its a 
 free action to un-hunker
 
+Stealth Attacks:
+If you are stealthed (undetected by an opponent) when you initiate an attack
+action, you incur bonuses depending on the circumstances of the attack.
+	Ranged weapon stealth-attacks (including explosive but not any other thrown 
+weapons) have normal damage with -1 miss chance, if 'adequate' aiming time is 
+given (DM's discretion: how rushed you are, or how skilled you are at stealth, etc).

```

Comment:
To my knowledge, there isn't any extra lethality to a ranged stealth kill? Do we normally apply some kind of bonus?
In one sense I improved the lethality by making it so you can land hits more easily.